src/lib/atoms/Heading/README.md

```diff
@@ -0,0 +1,5 @@
+# Heading
```

Comment:
This readme could probably better explain what this component is for; someone coming along to read this wouldn't necessarily know what the `tagname` stuff is about (since that's only relevant in testing)

The important detail is that this returns a `<h1>` to `<h6>` element, depending on the `level` prop passed in